src/components/duelling-picklist/index.ts

```diff
@@ -1,6 +1,16 @@
 export { default as DuellingPicklist } from "./duelling-picklist.component";
+export type { DuellingPicklistProps } from "./duelling-picklist.component";
+
 export { default as PicklistDivider } from "./picklist-divider/picklist-divider.component";
```

Comment:
I realise this is being quite pedantic, but because we're no longer exporting any `PicklistDividerProps`(because they don't exist - but they were previously exported from the `index.d.ts`) this could in theory break any implementations that were using styled-component props on this component. (This would have worked previously as well, but now it won't.) Maybe it's simpler to leave this export, and the original props interface, as it was, rather than label this a breaking change?